Nondeterministic test failure in test_einsum.py

I suspect this is due to nondeterminism somewhere, like use of a dict rather than an `OrderedDict`.
```
==== FAILURES ====
____ test_einsum[a,ab,bc,cd->] ____

equation = 'a,ab,bc,cd->'

    @pytest.mark.parametrize('equation', EINSUM_EXAMPLES + XFAIL_EINSUM_EXAMPLES)
    def test_einsum(equation):
        inputs, outputs, operands, sizes = make_example(equation)
        funsor_operands = [
            funsor.Tensor(operand, OrderedDict([(d, funsor.bint(sizes[d])) for d in inp]))
            for inp, operand in zip(inputs, operands)
        ]
        expected = torch.einsum(equation, operands)
        actual = naive_einsum(equation, *funsor_operands)
        assert expected.shape == actual.data.shape
>       assert torch.allclose(expected, actual.data)
E       AssertionError: assert False
E        +  where False = <built-in method allclose of type object at 0x11953f470>(tensor(-0.0174), tensor(-0.0174))
E        +    where <built-in method allclose of type object at 0x11953f470> = torch.allclose
E        +    and   tensor(-0.0174) = Tensor(-0.01743006706237793, OrderedDict(), 'real').data

test/test_einsum.py:76: AssertionError
==== 1 failed, 785 passed, 9 skipped, 16 xfailed in 3.28 seconds ====
make: *** [test] Error 1
```

--- a/test/conftest.py
+++ b/test/conftest.py
@@ -1,0 +1,7 @@
+from __future__ import absolute_import, division, print_function
+
+import pyro
+
+
+def pytest_runtest_setup(item):
+    pyro.set_rng_seed(0)

